README.md

```diff
@@ -172,10 +174,12 @@ There are 3 settings available at the moment:
  - **Hide Email Domain in Username**
    - This setting toggles the visibility of domain names in user names, if the user name is an email.
  - **Prompt for Checkout on Change**
-   - Unchecking this setting will make the Editor consider all files as already checked out. In that case, you won't get
+   - Un-checking this setting will make the Editor consider all files as already checked out. In that case, you won't get
      any notifications when you modify assets, and the "Checkout Assets" dialog won't show when you save those changes.
      This mimics how Git works, i.e. allowing the user to perform changes without worrying about checking out items.
      Note: Changelists don't currently support locally changed assets (ie not checked-out)
+ - **Limit Number of Revisions in History**
+   - If a non-null value is set, limit the maximum number of revisions requested to Plastic SCM to display in the "History" window.
```

Comment:
The behaviour of this new limit depends on the PlasticSCM version. It'd be useful to let users know (at least) about that minimum required version and the default limit of 50.